### PR TITLE
Add missing slash, to do like other examples

### PR DESCRIPTION
**Pull request by williamdes**
### Updated Docs
Screenshot 2023-10-12 at 12 08 25 PM

### CODE REVIEW

**Comment by williamdes** [comment]
Sorry to ping you @schew2381
You seem like a good reviewer for my change, what do you think about it?

**Comment by schew2381** [comment]
> Sorry to ping you @schew2381 You seem like a good reviewer for my change, what do you think about it?

Hi there, np for the ping and sorry for the delay! The change looks good to me 👍

**Review by schew2381 — APPROVED** [review]

**Comment by williamdes** [comment]
Should I rebase this one?

File: src/docs/integrations/slack/index.mdx [hunk 1]
```diff
@@ -108,7 +108,7 @@ Navigate to __Slash Commands__ under __Features__. Click __Create New Command__
 | Setting                         | Value                                         |
 | ------------------------------- | --------------------------------------------- |
 | Command                         | `/sentry`                                     |
-| Request URL                     | `{YOUR_DOMAIN}extensions/slack/commands/`     |
+| Request URL                     | `{YOUR_DOMAIN}/extensions/slack/commands/`     |
```
Comment (lizokm) [inline src/docs/integrations/slack/index.mdx:111]:
```suggestion
| Request URL                     | `{YOUR_DOMAIN}/extensions/slack/commands/` |
```

**Comment by schew2381** [comment]
> Should I rebase this one?

Should be good now, I think the spacing was just off haha. Thanks for the help!

**Pull request merged**